cmd/passthru/main.go

```diff
@@ -9,10 +9,13 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/zap",
+	"go.uber.org/zap/zapcore",
 	"github.com/gaukas/passthru/config"
 	"github.com/gaukas/passthru/handler"
 	"github.com/gaukas/passthru/protocol"
 	"github.com/gaukas/passthru/protocol/tls"
+	"github.com/gaukas/internal/logger"
```

Comment:
I think this line is an including error. It should be something like `./internal/logger` or `github.com/gaukas/passthru/internal/logger`